sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -2038,7 +2038,7 @@ await foreach (Page<BlobItem> page in test.Container.GetBlobsAsync(states: BlobS
             Assert.AreEqual(setMetadataResponse.Value.VersionId, blobs[1].VersionId);
         }
 
-        [PlaybackOnly("Object Replication policies is only enabled on certain storage accounts")]
+        [Ignore("Object Replication policies is only enabled on certain storage accounts")]
```

Comment:
what's the plan to bring back this test?

Reply:
See above.